packages/ember-metal/tests/computed_test.js

```diff
@@ -900,6 +900,16 @@ QUnit.test('is chainable', function() {
   ok(cp instanceof ComputedProperty);
 });
 
+QUnit.test('throws assertion if called over a CP with a setter', function() {
+  expectAssertion(function() {
+    computed({
+      get: function() { },
+      set: function() { }
+    }).readOnly();
+  }, /Computed properties that define a setter cannot be read-only/);
+  ok(true, 'No errors were thrown');
```

Comment:
The last `ok`? I've seen similar lines in other tests related to assertions, but I agree that is redundant. Gone